### Hi there ðŸ‘‹

<p>I'm studying Cellular and molecular biology at UniversitÃ© de Sherbrooke, in Canada. I have a bachelor's degree in Economics and I speak French, English and German. I am also a craft beer specialist and still have a foot in that sector in my hometown. </p>

<p>I'm currently doing an internship in Laforest-Lapointe Lab, where I'm learning to use different tools to process metagenomic (currently shotgun) data from samples to analysis. This lab deals in microbial ecology, so I apply these tools in the context of microbiome studies in humans and plants. One of my goals is to create pipelines suited to the needs of my colleagues' research and make them available to the public.</p>

<p>Since there are several available, seemingly substitutable tools for each single step of any metagenomic pipeline, I am also reviewing the literature to understand, and eventually advise on, which tool to use for any given step. Metagenomic research publications often lack a rationale behind the choice of tools, which suggests such a choice is often being made based on what others have done. In such a case, any bias introduced by a given tool spreads across the literature and a tool with a high citation rate may give a false impression of gold-standard.</p>
  
### A case for deeper benchmarking of bioinformatic tools accuracy
<p>As of 2022, many bioinformatic tools have been reviewed, tested and benchmarked; however, the focus is often on computational requirements. I want to find out which tools are the most accurate and what biases are being introduced by each approach. Although some benchmarking have produced such information for certain tools (especially assemblers and aligners), the answer is often non-triviial, especially with assembly-based (de novo) approaches. </p>

<p>Though it is possible that, one day, gold-standards be recognized in metagenomic bioinformatics, many parameters and tool choices seem to depend on data quality and origin, research objectives, and processing approach choices. Since the field of metagenomics is undergoing an extremely fast evolution, I expect that the best way to help researchers prepare their data for downstream analyses is not only to make the tools accessible, but to guide them in choosing the right tools and parameters thereof.</p> <p>In this regard, my goal is to summarize </p>
<ul>
  <li>which tools are better suited to any given data processing task;</li>
  <li>their function and crucial parameters; </li>
  <li>the reviews supporting their accuracy; </li>
  <li>articles supporting the choice of parameters; </li>
  <li>any recommended best practice for their use.</li>
</ul>

<p>I believe that such information will be valuable to any researcher who does not want to blindly rely on preconceived pipelines that may or may not suit their data and scientific objectives. I think any bioinformatic analysis should be understood and thought-through before being run, and therefore believe that a tool-intensive process such as metagenome sequencing data processing should be pondered and understood.</p>

<p>This is a work in progress in its early stage. My notes are mostly draft and I am getting acquainted with GitHub. More updates along the way!</p>
  
  
<!--
**jorondo1/jorondo1** is a âœ¨ _special_ âœ¨ repository because its `README.md` (this file) appears on your GitHub profile.

Here are some ideas to get you started:

- ðŸ”­ Iâ€™m currently working on ...
- ðŸŒ± Iâ€™m currently learning ...
- ðŸ‘¯ Iâ€™m looking to collaborate on ...
- ðŸ¤” Iâ€™m looking for help with ...
- ðŸ’¬ Ask me about ...
- ðŸ“« How to reach me: ...
- ðŸ˜„ Pronouns: ...
- âš¡ Fun fact: ...
-->
